### PR TITLE
Added condition to deal with empty rings

### PR DESCRIPTION
**Pull request by adamgoerdt**

### CODE REVIEW

File: pick.lic [hunk 1]
```diff
@@ -79,12 +79,17 @@ class LockPicker
         next
       end
 
-      until 'not even locked' == bput("pick my #{box} blind", 'not even locked', 'Roundtime')
```
Comment (rpherbig) [inline pick.lic]: `until` is a loop construct. By removing it, it appears that this code will no longer loop. Is that intentional?

Comment (adamgoerdt) [inline pick.lic]: Not intentional.  Non-looping bug has been corrected

File: pick.lic [hunk 2]
```diff
@@ -79,15 +79,26 @@ class LockPicker
...
-        waitrt?
-        next unless DRSkill.getxp('Locksmithing') >= 30
+      locked=true
+      while locked
+        case bput("pick my #{box} blind", 'not even locked', 'Roundtime','Find a more appropriate tool')
+        when 'not even locked'
+          waitrt?
+          locked=false
```
Comment (rpherbig) [inline pick.lic]: Is `locked=false` the same as `break`?

File: pick.lic [hunk 3]
```diff
@@ -79,15 +79,26 @@ class LockPicker
         next
       end
 
-      until 'not even locked' == bput("pick my #{box} blind", 'not even locked', 'Roundtime')
-        waitrt?
-        next unless DRSkill.getxp('Locksmithing') >= 30
```
Comment (rpherbig) [inline pick.lic]: This line appears to have checked an exp threshold. Without it, will the loop on line 83 continue even if you get above 30/34?

File: pick.lic (hunk 2 above)
Comment (adamgoerdt) [inline pick.lic]: They're effectively the same.  Is there a preferred style I should use?

Comment (rpherbig) [inline pick.lic]: If they are the same, I would prefer `break` so there is one less variable around.

File: pick.lic (hunk 3 above)
Comment (adamgoerdt) [inline pick.lic]: Whoops.  Looks like it will finish picking the current box even if locked.  Fix pushed

**Pull request merged**